dlt/destinations/impl/bigquery/factory.py

```diff
@@ -69,6 +71,27 @@ def ensure_supported_type(
                 "Enable `autodetect_schema` in config or via BigQuery adapter", column["data_type"]
             )
 
+    def to_db_datetime_type(
+        self,
+        column: TColumnSchema,
+        table: PreparedTableSchema = None,
+    ) -> str:
+        column_name = column["name"]
+        table_name = table["name"]
+        timezone = column.get("timezone", True)
+        precision = column.get("precision")
+
+        if precision and precision != self.capabilities.timestamp_precision:
+            logger.warn(
+                f"BigQuery does not support custom precision for column '{column_name}' in"
+                f" table '{table_name}'. Will use default precision."
+            )
+
+        # TIMESTAMP is always timezone-aware in BigQuery
+        # DATETIME is always timezone-naive in BigQuery
+        # NOTE: we disable DATETIME because it does not work with parquet
+        return "TIMESTAMP" if timezone else "TIMESTAMP"
```

Comment:
Is this line correct? The condition does not change anything..

Reply:
it is but I'll remove it to make it 100% clear. DATETIME does not work on bigquery in practice